LXReorderableCollectionViewFlowLayout/LXReorderableCollectionViewFlowLayout.m

```diff
@@ -258,7 +260,7 @@ - (void)handleScroll:(CADisplayLink *)displayLink {
                 distance = maxX - contentOffset.x;
             }
             
-            translation = CGPointMake(distance, 0.0f);
+            translation = CGPointMake(0.0f, 0.0f);
```

Comment:
These are actually made unnecessary by the restriction on `panTranslationInCollectionView` in `handlePanGesture`. I'll remove the changes.